src/options/view.rs

```diff
@@ -538,6 +541,8 @@ mod test {
         test!(relative:  TimeFormat <- ["--time-style", "relative"], None;  Both => like Ok(TimeFormat::Relative));
         test!(long_iso:  TimeFormat <- ["--time-style=long-iso"], None;     Both => like Ok(TimeFormat::LongISO));
         test!(full_iso:  TimeFormat <- ["--time-style", "full-iso"], None;  Both => like Ok(TimeFormat::FullISO));
+        test!(custom_style:  TimeFormat <- ["--time-style", "+%Y/%m/%d"], None;  Both => like Ok(TimeFormat::Custom { .. }));
+        test!(bad_custom_style:  TimeFormat <- ["--time-style", "%Y/%m/%d"], None;  Both => err OptionsError::BadArgument(&flags::TIME_STYLE, OsString::from("%Y/%m/%d") ));
```

Comment:
This part should be formatted similar to the other lines proceeding it, if rustfmt ignores this.